excel-importer/src/test/java/uk/gov/hmcts/ccd/definition/store/excel/endpoint/exception/SpreadsheetValidationErrorMessageCreatorTest.java

```diff
@@ -707,10 +708,10 @@ public void shouldHaveValidationMessageForCaseFieldEntityInvalidMetadataFieldVal
     }
 
     @Test
-    public void eventEntityInvalidUserRole_createErrorMessageCalled_customMessageReturned_whenIdamRoleMissing() {
+    public void eventEntityInvalidUserRole_createErrorMessageCalled_customMessageReturned_whenIdamAPMissing() {
```

Comment:
Rename test when validation error `EventEntityInvalidUserRoleValidationError` is renamed

Reply:
Fixed in 3951e2b